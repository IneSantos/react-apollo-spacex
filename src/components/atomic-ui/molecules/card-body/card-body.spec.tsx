import React from "react";
import { render } from "@testing-library/react";
import CardBody from "./card-body";

describe("CardBody molecule tests", () => {
  test(`CardBody component render`, () => {
    const { container } = render(<CardBody details="This is a detail" link="This is a link"/>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
