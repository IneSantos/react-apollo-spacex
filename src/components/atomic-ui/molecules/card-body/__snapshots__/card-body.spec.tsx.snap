// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardBody molecule tests CardBody component render 1`] = `
<div
  class="card-body"
>
  <div
    class="details"
  >
    This is a detail
  </div>
  <a
    class="card-action "
    href="This is a link"
  >
     
    See more
     
  </a>
</div>
`;
